### ISSUE TITLE
can't find 'hdf5_hl_cpp.lib'

### ISSUE THREAD

**Issue by thomassligon** [ICB-DCM/AMICI#1169]
I'm running
```
importer = pypesto.petab.PetabImporter(petab_problem)
model = importer.create_model()
```
which leads to an attempt to create the model using MSVC, including link.exe and linking the following objects:
`amici.lib sundials.lib suitesparse.lib hdf5_hl_cpp.lib hdf5_hl.lib hdf5_cpp.lib hdf5.lib`
but it causes
`LINK : fatal error LNK1181: cannot open input file 'hdf5_hl_cpp.lib'`
hdf5 is NOT enabled.
I can supply more details if necessary.

**Comment by dweindl** [comment]
What's the output of `from amici.setuptools import get_hdf5_config; get_hdf5_config()`? (In case that fails with above error, copy the amici `setuptools.py` to a different directory and import from there directly.)

**Comment by thomassligon** [comment]
I ran
```
from amici.setuptools import get_hdf5_config
out = get_hdf5_config()
print(out)
```
and the result is
`{'include_dirs': [], 'library_dirs': [], 'libraries': [], 'define_macros': [], 'found': False}`

**Comment by dweindl** [comment]
Okay, that's what I would expect. No clue where the hdf5 dependency would come from...

Do you have a complete installation log, including the compiler/linker invocation?

**Comment by thomassligon** [comment]
I have attached what I saved from the output. If you need more, I can try to run the whole thing in verbose mode. 
[out-2020-07-23.docx](https://github.com/ICB-DCM/AMICI/files/4970939/out-2020-07-23.docx)

**Comment by dweindl** [comment]
Ah... is it possible, that there is still an old `amici.py` lying around in your amici installation directory from a previous installation? You can delete that file and retry.

**Comment by dweindl** [comment]
@thomassligon : problem solved with v0.11.4?

**Comment by thomassligon** [comment]
Yes, this is solved with amici v0.11.4
